### DISCUSSION TITLE
Combine multiple repositories for one project

### DISCUSSION THREAD

**Discussion by mtcn** [codemagic-ci-cd/codemagic-docs#2169] [Q&A] [answered]
### Use case

```Text
I have an iOS project that consists of two modules residing in different repositories. I need to connect other repositories to Codemagic.
```

### Idea

```Text
I'm not sure if this feature exists but I would like to connect other dependent repositories to Codemagic to get build
```

**Comment by Ycoi** [comment, marked as answer]
Yes, this is already possible. You can achieve this for example by using git submodules. Please check [our docs](https://docs.codemagic.io/knowledge-git/access-private-git-submodules/) for more information on how to do this.

Since this is not in effect a feature request, I’ll move this discussion to the Q&A section, there you can ask further help from our support engineers if you need or encounter issues when setting this up.

**Reply by mtcn** [reply]
thanks, I will try it out
